components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2189,8 +2191,21 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
                 throw new Error(`No Stripe customer profile for '${attributionId}'`);
             }
 
-            await this.stripeService.setDefaultPaymentMethodForCustomer(customerId, setupIntentId);
-            await this.stripeService.createSubscriptionForCustomer(customerId, attributionId);
+            const createStripeSubscriptionOnUsage = await getExperimentsClientForBackend().getValueAsync(
+                "createStripeSubscriptionOnUsage",
+                false,
+                {
+                    user: user,
+                    teamId: team ? team.id : undefined,
+                },
+            );
+
+            if (createStripeSubscriptionOnUsage) {
+                await this.billingService.createStripeSubscription({ attributionId, setupIntentId, usageLimit });
+            } else {
+                await this.stripeService.setDefaultPaymentMethodForCustomer(customerId, setupIntentId);
+                await this.stripeService.createSubscriptionForCustomer(customerId, attributionId);
+            }
 
             // Creating a cost center for this customer
             const { costCenter } = await this.usageService.setCostCenter({
```

Comment:
As a follow-up, we should then move the cost-center setting into the `CreateStripeSubscription` call